lib/primo/query.rb

```diff
@@ -74,7 +74,7 @@ def exclude_facets
   end
 
   private
-    REQUIRED_PARAMS = [ :field, :precision, :value ]
```

Comment:
Does this mean there is no way to override the precision on a per case basis?

Reply:
No, it means that if you omit precision, the default, `:contains` will be used. Currently, if you omit precision, it raises an error.